### PR TITLE
Add React keys in subscribe-channels (issue #8729)

### LINKED ISSUE #8729: Missing React key in software channels page
In development mode e.g. https://localhost:8080/rhn/systems/details/SystemChannels.do?sid=1000010006& throws

```
subscribe-channels.tsx:600 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `SystemChannels` ...
```

### PR DESCRIPTION
**Pull request by jhonnyV-V**
## What does this PR change?

Add a key property in multiple elements, this fixes #8729  

## GUI diff

No difference.

## Documentation
- No documentation needed

## Test coverage
- No tests: already covered

## Links

Issue(s): #8729

## Changelogs
If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

### CODE REVIEW

**Comment by Etheryte** [comment]
Thanks a lot for stopping by and taking the time to fix this, that's very kind of you. I have one small nitpick about the type annotation, but outside of that this looks good. Could you please run `yarn lint` to fix the premerge checks?

File: web/html/src/manager/systems/ssm/ssm-subscribe-channels.tsx
Comment (Etheryte) [inline web/html/src/manager/systems/ssm/ssm-subscribe-channels.tsx:409]: Here and below, this manual annotation is not really necessary since the type is already inferred from the prop type.

**Comment by jhonnyV-V** [comment]
> Thanks a lot for stopping by and taking the time to fix this, that's very kind of you. I have one small nitpick about the type annotation, but outside of that this looks good. 👍
> 
> Could you please run `yarn lint` to fix the premerge checks?

sure, I just did the linting and removed the type annotation, I squashed the commits and force pushed to keep the history clean

**Review by Etheryte — APPROVED** [review]
Thanks, much appreciated. That looks good, approved.

Fyi you will not be able to merge this PR yourself, I will merge it once our release cycle has branches open again.

**Comment by deneb-alpha** [comment]
failures on tests are unrelated. checked with QE  team before merging

**Pull request merged**
